packages/design-system/src/components/DateField/SingleInputDateField.stories.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import SingleInputDateField from './SingleInputDateField';
+
+export default {
+  title: 'Components/SingleInputDateField',
+  component: SingleInputDateField,
+  argTypes: {
+    errorMessage: {
+      control: { type: 'text' },
+    },
+    hint: {
+      control: { type: 'text' },
+    },
+    label: {
+      control: { type: 'text' },
+    },
+    requirementLabel: {
+      control: { type: 'text' },
+    },
+  },
+  args: {
+    label: 'Birthday',
+    hint: 'Please enter your birthday',
+    name: 'single-input-date-field',
+  },
+};
+
+const Template = ({ ...args }) => {
+  const [dateString, setDateString] = useState('');
+  return <SingleInputDateField {...args} value={dateString} onChange={setDateString} />;
+};
+
+export const Default = Template.bind({});
+
+export const WithPicker = Template.bind({});
+WithPicker.args = {
```

Comment:
Is the styling for this component complete? Because in Storybook I'm not seeing an outline around the calendar popup like in the mocks
Storybook:
<img width="343" alt="Screen Shot 2022-06-01 at 10 06 35 AM" src="https://user-images.githubusercontent.com/33579665/171450119-48a4d56b-cd6c-4cff-9341-56755e968490.png">

Mocks:
<img width="343" alt="Screen Shot 2022-06-01 at 10 07 04 AM" src="https://user-images.githubusercontent.com/33579665/171450163-331cee5f-55bc-42c9-a4c6-8924f031bbc3.png">